changelog/865.feature.rst

```diff
@@ -0,0 +1,25 @@
+.. Added aliases (with trailing underscores) for parameters in formulary::
```

Comment:
```suggestion
Added aliases (with trailing underscores) for parameters in the formulary:
```

Reply:
Primarily, the `..` and `::` don't translate to a clean reST file by `towncrier`.